### PR TITLE
Add an eval() function in the MultimodalModel class 

### PR DESCRIPTION
**Pull request by MinkyoungCho**
This enables `model.eval()` execution, which is necessary to remove random elements (e.g., dropout) during the validation process.

Previously, even though we attempted to set the model to evaluation mode, it would still call `train()` and throw an error due to the missing `eval()` function. By adding the `eval()` function (see the code below), we can disable training and run without error.

```
def eval(self):
        super().train(False)
```

image

### CODE REVIEW

**Review by insujang — APPROVED** [review]
Thank you for your contribution!
